_config.yml

```diff
@@ -11,7 +11,7 @@ owner:
   name: Ash Wilson
   avatar: cat-sniffer.jpg
   bio: >
-    Ash Wilson is a Developer Support Engineer at Rackspace, a Ruby hacker, and a Dad.
+    In between DadOps, Ash Wilson writes Go Go Gadgets as a Software Engineer at Rackspace.
```

Comment:
Hah, nah.

Let me think of something else to put here. I should also change that "tagline" although I'm not sure where it shows up.